ranked-choice-voting/generate_condorcet_winner.py

```diff
@@ -111,6 +111,5 @@ def condorcet_main(file_path_of_election):
 
 
 if __name__ == "__main__":  # # would read in concatenated csvs here C:\Users\anxhe\Github\ranked-choice-voting\data\election_09-22-2020_23-00-57_3cands_0.02666666666666667noise.csv
-    df = condorcet_main('data/election_09-22-2020_23-00-57_3cands_0.02666666666666667noise.csv')
-
+    df = condorcet_main('../data13/election_09-22-2020_23-00-57_3cands_0.02666666666666667noise.csv')
```

Comment:
Thought for the future: we should have a "fixures" directory where we can store 2-3 CSVs for unit testing & running our code